firefox-ios/Client/Frontend/Settings/AppIconSelection/AppIconView.swift

```diff
@@ -0,0 +1,84 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import SwiftUI
+import Common
+
+struct AppIconView: View, ThemeApplicable {
+    let appIcon: AppIcon
+    let isSelected: Bool
+    let windowUUID: WindowUUID
+
+    let setAppIcon: (AppIcon) -> Void
+
+    // MARK: - Theming
+    // FIXME FXIOS-11472 Improve our SwiftUI theming
+    @Environment(\.themeManager)
+    var themeManager
```

Comment:
```suggestion
    @Environment(\.themeManager) var themeManager
```
nit: indent to match with how other property wrappers are written